pkg/remotewrite/remotewrite.go

```diff
@@ -311,6 +318,7 @@ func (swm seriesWithMeasure) MapPrompb() []*prompb.TimeSeries {
 		}
 	}
 
+	//nolint:forcetypeassert
```

Comment:
What is the error?

Reply:
It complains that the type casts are not checked. Which is indeed the case. The method doesn't return an error, so better safe than sorry I chose to keep the current behavior (panic) in case of issue, and deactivate the linter check locally.